src/libs/connectors/sendConnectorsLogs.spec.js

```diff
@@ -0,0 +1,177 @@
+import CozyClient from 'cozy-client'
```

Comment:
This whole file should be in .ts (new file + tested file is written in ts)

Reply:
As discussed we @Ldoppea and you, we will keep it in JS, too much difficulty to made a clean cozy-client mock in .ts